launcher/minecraft/MojangVersionFormat.cpp

```diff
@@ -306,6 +306,15 @@ void MojangVersionFormat::writeVersionProperties(const VersionFile* in, QJsonObj
         }
         out.insert("downloads", downloadsOut);
     }
+    if(in->compatibleJavaMajors.size())
```

Comment:
```suggestion
    if(!in->compatibleJavaMajors.isEmpty())
```

i know the code above this uses `size()`, but that is bad anyway, we should use the correct thing on new code :p